#4463 FAQ for Maven compilation failure with error like python2 not found (#4473)
apache/skywalking: docs/en/FAQ/maven-compile-npm-failure.md

[ERROR] gyp ERR! stack 
[ERROR] gyp ERR! stack     at ChildProcess.exithandler (child_process.js:275:12)
[ERROR] gyp ERR! stack     at emitTwo (events.js:126:13)
[ERROR] gyp ERR! stack     at ChildProcess.emit (events.js:214:7)
[ERROR] gyp ERR! stack     at maybeClose (internal/child_process.js:925:16)
[ERROR] gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:209:5)
[ERROR] gyp ERR! System Windows_NT 10.0.17134
......
[INFO] server-starter-es7 ................................. SUCCESS [ 11.657 s]
[INFO] apm-webapp ......................................... FAILURE [ 25.857 s]
[INFO] apache-skywalking-apm .............................. SKIPPED
[INFO] apache-skywalking-apm-es7 .......................... SKIPPED
```

### Reason

It has nothing to do with skywalking.   
According to https://github.com/sass/node-sass/issues/1176, if you live in countries where requesting resources from `github` and `npmjs.org` is very slowly, some precompiled binaries for dependency `node-sass` will fail to be downloaded during `npm install`, then npm will try to compile them itself. That's why `python2` is needed.

### Resolve

> #### 1. If you live in China, please edit `skywalking\apm-webapp\pom.xml`     

Reviewer: ```suggestion
#### 1. Use mirror. Such as in China, please edit `skywalking\apm-webapp\pom.xml`     
```

Find
```
<configuration>  
 <arguments>install --registry=https://registry.npmjs.org/</arguments>  
</configuration>
```
Replace it with
```
<configuration>  
 <arguments>install --registry=https://registry.npm.taobao.org/ --sass_binary_site=https://npm.taobao.org/mirrors/node-sass/</arguments>  
</configuration>
```

> #### 2. Get an enough powerful VPN

Reviewer: ```suggestion
#### 2. Use a better network.
```
